src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Apprentice/EditApprenticeshipViewModelToValidateApprenticeshipForEditMapper.cs

```diff
@@ -25,7 +25,9 @@ public Task<ValidateApprenticeshipForEditRequest> Map(EditApprenticeshipRequestV
                 TrainingCode = source.CourseCode,
                 ProviderId = source.ProviderId,
                 Version = source.Version,
-                Option = source.Option == "TBC" ? string.Empty : source.Option
+                Option = source.Option == "TBC" ? string.Empty : source.Option,
+                EmploymentEndDate = source.EmploymentEndDate.Date,
+                EmploymentPrice = source.EmploymentPrice
```

Comment:
I like to add the trailing comma as it prevents future diffs from highlighting this line (which won't have materially changed)